src/components/Fastlege.tsx

```diff
@@ -43,12 +43,12 @@ interface FastlegeProps {
 }
 
 export const Fastlege = ({ fnr }: FastlegeProps) => {
-  const { isLoading, isError, error, data } = useFastlegeQuery(fnr);
+  const { isInitialLoading, isError, error, data } = useFastlegeQuery(fnr);
```

Comment:
Endret til isInitialLoading for queries som kan være disabled (Ref: https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-4#disabled-queries https://tanstack.com/query/v4/docs/react/guides/disabling-queries#isinitialloading)